addon/components/frost-radio-button.js

```diff
@@ -46,5 +46,27 @@ export default Component.extend({
       `${this.toString()} must be initialized with a 'value' property`,
       this.get('value')
     )
+  },
+  keyPress (e) {
+    e.preventDefault()
+    e.stopPropagation()
+
+    if (this.get('disabled') || this.get('groupValue') === this.get('value')) {
+      return
+    }
+
+    let event = Ember.$.Event(null, e)
+    let target = Ember.$.clone(Ember.$(e.target).find('input')[0])
+    let change = this.get('onChange')
```

Comment:
Might as well move all of these inside the keyCode check - also, probably worthwhile to package this up into a reusable function since we call it in two places